plugins/module_utils/network/ios/rm_templates/snmp_server.py

```diff
@@ -0,0 +1,1720 @@
+                ^snmp-server\senable\straps\sconfig
+                """, re.VERBOSE),
+            "setval": "snmp-server enable traps config",
+            "result": {
+                "traps": {
+                    "config": True,
+                },
+            },
+        },
+        {
+            "name": "traps.isis",
+            "getval": re.compile(
+                r"""
+                ^snmp-server\senable\straps\sisis$
+                """, re.VERBOSE),
+            "setval": "snmp-server enable traps isis",
+            "result": {
+                "traps": {
+                    "isis": True,
+                },
+            },
+        },
+        {
+            "name": "traps.config_copy",
+            "getval": re.compile(
+                r"""
+                ^snmp-server\senable\straps\sconfig-copy
+                """, re.VERBOSE),
+            "setval": "snmp-server enable traps config-copy",
+            "result": {
+                "traps": {
+                    "config_copy": True,
+                },
+            },
+        },
+        {
+            "name": "traps.config_ctid",
+            "getval": re.compile(
+                r"""
+                ^snmp-server\senable\straps\sconfig-ctid
+                """, re.VERBOSE),
+            "setval": "snmp-server enable traps config-ctid",
+            "result": {
+                "traps": {
+                    "config_ctid": True,
+                },
+            },
+        },
+        {
+            "name": "traps.dhcp",
+            "getval": re.compile(
+                r"""
+                ^snmp-server\senable\straps\sdhcp
+                """, re.VERBOSE),
+            "setval": "snmp-server enable traps dhcp",
+            "result": {
+                "traps": {
+                    "dhcp": True,
+                },
+            },
+        },
+        {
+            "name": "traps.eigrp",
+            "getval": re.compile(
+                r"""
+                ^snmp-server\senable\straps\seigrp
+                """, re.VERBOSE),
+            "setval": "snmp-server enable traps eigrp",
+            "result": {
+                "traps": {
+                    "eigrp": True,
+                },
+            },
+        },
+        {
+            "name": "traps.entity",
+            "getval": re.compile(
+                r"""
+                ^snmp-server\senable\straps\sentity
+                """, re.VERBOSE),
+            "setval": "snmp-server enable traps entity",
+            "result": {
+                "traps": {
+                    "entity": True,
+                },
+            },
+        },
+        {
+            "name": "traps.energywise",
+            "getval": re.compile(
+                r"""
+                ^snmp-server\senable\straps\senergywise$
+                """, re.VERBOSE),
+            "setval": "snmp-server enable traps energywise",
+            "result": {
+                "traps": {
+                    "energywise": True,
```

Comment:
is this always True?

Reply:
yes, if the parser hits a match it is supposed to be true.